### PR TITLE
Move to the forked Sobek code

### PR DESCRIPTION
**Pull request by mstoykov**
## What?

Move to Grafana fork of goja called Sobek - now with the actual code not just aliases

## Why?

See #3772 

## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [x] I have run linter locally (`make lint`) and all checks pass.
- [x] I have run tests locally (`make tests`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Review by codebien — DISMISSED** [review]

**Review by oleiade — DISMISSED** [review]
🎉 🚀

**Review by codebien — DISMISSED** [review]

File: cmd/runtime_options.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func runtimeOptionFlagSet(includeSysEnv bool) *pflag.FlagSet {
 	flags.Bool("include-system-env-vars", includeSysEnv, "pass the real system environment variables to the runtime")
 	flags.String("compatibility-mode", "extended",
 		`JavaScript compiler compatibility mode, "extended" or "base" or "experimental_enhanced"
-base: pure goja - Golang JS VM supporting ES5.1+
+base: pure sobek - Golang JS VM supporting ES5.1+
```
Comment (codebien) [inline cmd/runtime_options.go:27]:
```suggestion
base: pure Sobek - Golang JS VM supporting ES5.1+
```
Can we have it with a capitalized letter? I see this across multiple lines across this pull request. Consider this suggestion for all of them.

Comment (mstoykov) [reply cmd/runtime_options.go:27]: I am not strictly against this, but we do not capitalize it anywhere else as well as far as I can see - and we do not really do it for our other projects.

Comment (codebien) [reply cmd/runtime_options.go:27]: You did it for the first file `Dependencies` :sweat_smile: 

Btw, I don't think we have other projects with a name independent from `k6`. All repositories are prefixed by `k6` or some combination of it like `xk6` where doesn't make sense to capitalize. But in this case, it is a proper name, so I don't see why we shouldn't as it is the correct grammar.

Of course, I expect this only for text and not for code (e.g repo, package names, etc..)

Comment (mstoykov) [reply cmd/runtime_options.go:27]: I made that lower case as well :sweat_smile:

**Review by codebien — APPROVED** [review]

**Review by oleiade — APPROVED** [review]

**Pull request merged**